cluster/gce/config-default.sh

```diff
@@ -549,3 +549,10 @@ export TLS_CIPHER_SUITES=""
 # CLOUD_PROVIDER_FLAG defines the cloud-provider value presented to KCM, apiserver,
 # and kubelet
 export CLOUD_PROVIDER_FLAG="${CLOUD_PROVIDER_FLAG:-gce}"
+
+# When ENABLE_AUTH_PROVIDER_GCP is set, following flags for out-of-tree credential provider for GCP
```

Comment:
Consider to also document that during node bootstrap, we will download the auth provider for GCP if this is enabled